testsite/bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "testsite",
+  "version": "1.0.1",
+  "homepage": "https://github.com/counsyl/civet",
+  "authors": [
+    "opensource@counsyl.com"
+  ],
+  "description": "Civet test app",
+  "license": "Apache License 2.0",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "jquery": "~2.1.3",
+    "bootstrap-sass-official": "~3.3.1",
+    "test": "aristretto/test"
```

Comment:
woops, it was marked as private, but since I was already authed it worked.  I made that package public, try again?